cmd/pebble/cmd_validate_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2022 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	. "gopkg.in/check.v1"
+
+	pebble "github.com/canonical/pebble/cmd/pebble"
+)
+
+func (s *PebbleSuite) TestValidateCommandYamlOK(c *C) {
```

Comment:
I think name is overly verbose: it doesn't need "command" in it because we're in the `cmd/pebble` directory so all these tests are about commands. Also `Yaml` is unnecessary (and would be capitalized as `YAML` in Go style). So I'd go with `TestValidateOK`.